hail/python/hail/utils/table_iterator.py

```diff
@@ -0,0 +1,38 @@
+from typing import Optional
+import hail as hl
+from concurrent.futures import ProcessPoolExecutor
+
+
+ht: Optional[hl.Table] = None
+
+
+def init_process(path):
+    global ht
+    assert ht is None
+    hl.init()
```

Comment:
Are there any non-default `init` parameters we'd want to consider here? I'm worried about log spam.